client/extensions/woocommerce/app/products/product-header.js

```diff
@@ -13,21 +13,26 @@ import ActionHeader from 'woocommerce/components/action-header';
 import Button from 'components/button';
 import { getLink } from 'woocommerce/lib/nav-utils';
 
-const ProductHeader = ( { onTrash, onSave, isBusy, translate, site, product } ) => {
+const ProductHeader = ( { onView, onTrash, onSave, isBusy, translate, site, product } ) => {
+	const existing = product && ! isObject( product.id );
+
+	const viewButton = onView &&
+		<Button borderless onClick={ onView }><Gridicon icon="visible" /> { translate( 'View' ) } </Button>;
+
 	const trashButton = onTrash &&
-		<Button borderless onClick={ onTrash }><Gridicon icon="trash" /></Button>;
+		<Button borderless scary onClick={ onTrash }><Gridicon icon="trash" /> { translate( 'Trash' ) } </Button>;
 
 	const saveExists = 'undefined' !== typeof onSave;
 	const saveDisabled = false === onSave;
 
+	const saveLabel = ( product && existing ? translate( 'Update' ) : translate( 'Save & Publish' ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **67** times:
`translate( 'Save &amp; Publish' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).